### PR TITLE
Debian release artifact

### PR DESCRIPTION
**Pull request by mattnibs**
Add `--linux` option to scripts/release. Currently will generate only debian
package (future prs will generate packages for rpm and flatpak).
Also add github action for creating linux release artifacts.

New dependency `electron-installer-debian` when imported as a devDependency
caused windows to fail on `npm i`. As a result I had to make it an optionalDependencies. 

AppIcon appears to be working (debian 10 running gnome 3):
image

part of #618

### CODE REVIEW

**Comment by alfred-landrum** [comment]
I was able to build & install the debian package (via `dpkg -i brim_amd64.deb`) and run it on a Ubuntu 18.04.1 LTS vm, including open a pcap & launching wireshark.
Screen Shot 2020-04-19 at 19 57 45

File: scripts/release/install.js [hunk 1]
```diff
@@ -62,6 +63,23 @@ module.exports = {
         // up exception may contain a passphrase.
         console.log("Error building win32 installer")
       })
+  },
+
+  debian: function() {
+    console.log("Building installer for debian")
```
Comment (alfred-landrum) [inline scripts/release/install.js:69]:
```suggestion
    console.log("Building deb package installer")
```

File: .github/workflows/linux-release.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+    - name: Set NPM Cache Directory
+      id: set-npm-cache-dir
+      run: echo "::set-output name=npm_cache_dir::$(npm config get cache)"
+    - name: Clear Extraneous Runner Cache
+      run: |
+          cachedir=${{ steps.set-npm-cache-dir.outputs.npm_cache_dir }}
+          rm -rf "${cachedir:?}"
```
[the remark this answers is not in the document]
Comment (mattnibs) [reply .github/workflows/linux-release.yml:23]: Sure

File: scripts/release/install.js (hunk 1 above)
Comment (alfred-landrum) [inline scripts/release/install.js:69]: You resolved my comment but did not make the suggested change?

Comment (mattnibs) [reply scripts/release/install.js:69]: I committed your suggestion but it got wiped out by my force push.

Comment (alfred-landrum) [reply scripts/release/install.js:69]: oops! :)

**Pull request merged**